### DISCUSSION TITLE
Proof of our academic status upload issue when requesting for student benefits

### DISCUSSION THREAD

**Discussion by Lishan997** [community/community#37249] [GitHub Education]
When I request for Git hub student pack , there is a section for upload proof of our academic status, I have my University ID, I took different photos from my android phone and tried to upload but there is a message "File is too small", then I tried Using webcam , but my webcam is not much quality but I took photo but after took photo there is a success message but no way to continue the process, Please can any one tell me alternative solution for this matter

**Comment by tawfik-s** [comment]
I have a similar issue with GitHub 
I use a phone camera which is high quality
make photo jpg photo instead of png 
try to not transfer photos to a computer using tools that will not reduce the photo quality
the university ID and data are in English if not translated it and add the translation using tools will not reduce the quality

i hope it will be verified

**Comment by rskbansal** [comment]
If webcam doesn't work for you, maybe try simulating a camera like `OBS Virtual Camera`, I did my verification using the same by creating a `media source` and adding my proof in the scene, then captured the photo when prompted for :)
